File: src/main/java/br/com/jiratorio/domain/request/LeadTimeConfigRequest.java
package br.com.jiratorio.domain.request;

import javax.validation.constraints.NotBlank;
import lombok.Data;

@Data
public class LeadTimeConfigRequest {

    @NotBlank
    private String name;

    @NotBlank
    private String startColumn;

    @NotBlank
    private String endColumn;

}

File: src/main/java/br/com/jiratorio/domain/request/LeadTimeConfigRequest.kt
package br.com.jiratorio.domain.request

import javax.validation.constraints.NotBlank

class LeadTimeConfigRequest {

    @NotBlank
    var name: String? = null

    @NotBlank
    var startColumn: String? = null

    @NotBlank
    var endColumn: String? = null

}
